api/axis.py

```diff
@@ -91,6 +95,43 @@ class Axis360API(BaseAxis360API, Authenticator, BaseCirculationAPI):
         (pdf, adobe_drm): 'PDF',
     }
 
+    def external_integration(self, _db):
+        return self.collection.external_integration
+
+    def _run_self_tests(self, _db):
+        result = self.run_test(
+            "Refreshing bearer token", self.refresh_bearer_token
+        )
+        yield result
+        if not result.success:
+            # If we can't get a bearer token, there's no point running
+            # the rest of the tests.
+            return
+
+        def _count_events():
+            now = datetime.utcnow()
```

Comment:
I think this one is missing `.datetime` in there?

Reply:
axis.py does `from datetime import datetime` instead of `import datetime`, so `datetime.utcnow()` is correct here, although it would be better to change the import to the way it's used elsewhere.